choropleth-us/choropleth.js

```diff
@@ -0,0 +1,97 @@
+import * as d3 from 'd3';
+import * as gChartcolour from 'g-chartcolour';
+import topojson from 'topojson-client';
+
+export function draw() {
+    // let interpolation =d3.curveLinear
+    let colourScale = d3.scaleThreshold();
+    let level = 'counties';
+    let rem = 16;
+    var projection = d3.geoIdentity()
```

Comment:
Change to `const`

Reply:
Doesn't that mean they can't be changed by the getter/setters?

Reply:
@bobhaslett Ah, right, apologies — I mean `let`.